Add custom export for molecule table

beckyjackson opened the pull request:
This new `molecule_export.tsv` will be packaged in the zip we include with each release. It contains the following:

* MRO ID
* Label
* IEDB Label
* Synonyms (separated by pipe)
* Parent
* In Taxon
* In Taxon ID

I was originally going to use `gizmos.export` for this, but with the "In Taxon" restriction, this took almost 10 minutes, so it just seemed better to use the existing TSVs - which is pretty much instant. That said, I kept the indexes because we'll probably need them anyway.

Resolves #99

jgbaum reviewed (commented):
The field list looks good to me!

beckyjackson commented:
@jgbaum here's the file itself, if you don't mind taking a look and making sure this looks good:
[molecule_export.tsv.gz](https://github.com/IEDB/MRO/files/6711841/molecule_export.tsv.gz)

We plan to include this file in the `iedb.zip` product we create with each MRO release. Does that work for you?

jgbaum commented:
Yes, it's beautiful!  @khaeuk, is there anything missing in this export that you would like for the allele validator / autocomplete?

khaeuk commented:
It would be nice if we can add another column that indicates if it's from 'mhci', 'mhcii', etc. Basically the "parent" column from the original molecule file. Other than that it looks good.
@jgbaum @beckyjackson

beckyjackson commented:
OK - I changed the "Parent" column to be the parent label, and added the "Parent ID" column for the MRO ID. @jgbaum @khaeuk please see attached file and let me know if this is what you need!

[molecule_export.tsv.gz](https://github.com/IEDB/MRO/files/6728167/molecule_export.tsv.gz)

khaeuk commented:
Thank you @beckyjackson 
I think it looks good to me.

khaeuk commented:
@beckyjackson Where will this file live? I would like to know where I can find it so that I can pull the data regularly. Thanks!

beckyjackson commented:
@khaeuk you can get it from the `iedb.zip` asset from the latest release - it's called `molecule_export.tsv` in that folder. We'll hopefully have a better URL for you to directly download that from soon.

jamesaoverton commented:
@beckyjackson I think it would be better to attach `molecule_export.tsv` as a separate file (not inside the zip file) to the release, so it's easier to fetch. It's a bit more work for us but easier for everybody else.
